release/tools/vmimage.subr

```diff
@@ -285,7 +285,7 @@ vm_create_disk() {
 			ROOTFSPART="-p ${FSPARTTYPE}/rootfs:=${VMBASE}"
 			MAKEFSARGS="$MAKEFSARGS -B little"
 			;;
-		arm:armv7 | arm64:aarch64 | riscv:riscv64*)
+		arm:armv7 | arm64:aarch64* | riscv:riscv64*)
```

Comment:
Upstream the riscv one is a hold-over from riscv64sf that I missed when removing sf support.  Fine to leave it upstream though. :)

Reply:
I figured. :)